services/galley/src/Galley/Validation.hs

```diff
@@ -49,29 +42,17 @@ rangeCheckedMaybe (Just a) = Just <$> rangeChecked a
 {-# INLINE rangeCheckedMaybe #-}
 
 -- Between 0 and (setMaxConvSize - 1)
-newtype ConvSizeChecked a = ConvSizeChecked {fromConvSize :: a}
-  deriving (Functor)
+newtype ConvSizeChecked f a = ConvSizeChecked {fromConvSize :: f a}
+  deriving (Functor, Foldable, Traversable)
 
--- Between 1 and setMaxConvSize
-data ConvMemberAddSizeChecked = ConvMemberAddSizeChecked {sizeCheckedLocals :: [(UserId, RoleName)], sizeCheckedRemotes :: [(Remote UserId, RoleName)]}
-
-checkedConvSize :: Bounds a => a -> Galley (ConvSizeChecked a)
+checkedConvSize :: Foldable f => f a -> Galley (ConvSizeChecked f a)
 checkedConvSize x = do
   o <- view options
   let minV :: Integer = 0
```

Comment:
You can just delete `minV` as it's not used anymore, can't you?

Reply:
It's still used in the error message.